### ISSUE TITLE
verbose errors: handler file missing

### ISSUE THREAD

**Issue by SET001** [ConduitVC/aws-utils#49]
If serverless.yml points to handler file which does not exist, all you will get is error like this:

```
✖ error Query.getReservations failed
✖ error undefined
```

There is no way to understand what is going wrong here. Can it be more verbose?

**Comment by cbaron** [comment]
Of course!  I encourage you to submit a PR, I will gladly assist along the way.

**Comment by SET001** [comment]
I would be able to offer a PR if you can point a place in code where attempt to read from handler file is made

**Comment by cbaron** [comment]
https://github.com/ConduitVC/aws-utils/blob/71ae2f3774540b425b70fb56a2836bf0b963f75d/packages/appsync-emulator-serverless/lambdaSource.js#L31
